DomainContent/Rust/DJ-Tools/DJ-Tools_Spawner.js

```diff
@@ -0,0 +1,560 @@
+                "x": "0.5",
+                "y": "0.5",
+                "z": "0.5"
+            },
+            "emitOrientation": {
+                "x": "-90.01",
+                "y": "0",
+                "z": "0"
+            },
+            "emitterShouldTrail": true,
+            "particleRadius": "0",
+            "radiusSpread": "4",
+            "radiusStart": "0.5799999833106995",
+            "radiusFinish": "0",
+            "color": {
+                "red": "255",
+                "blue": "255",
+                "green": "255"
+            },
+            "colorSpread": {
+                "red": "0",
+                "blue": "0",
+                "green": "0"
+            },
+            "colorStart": {
+                "red": "255",
+                "blue": "33",
+                "green": "33"
+            },
+            "colorFinish": {
+                "red": "239",
+                "blue": "255",
+                "green": "13"
+            },
+            "emitAcceleration": {
+                "x": "0.01",
+                "y": "0.01",
+                "z": "0.01"
+            },
+            "accelerationSpread": {
+                "x": "1",
+                "y": "1",
+                "z": "1"
+            },
+            "alpha": "0.6000000238418579",
+            "alphaSpread": "0",
+            "alphaStart": "0.09000000357627869",
+            "alphaFinish": "0",
+            "polarStart": "0",
+            "polarFinish": "1.02974",
+            "azimuthStart": "-180.00000500895632",
+            "azimuthFinish": "180.00000500895632"
+        },
+        lightBaseProps = {
+            type: "Light",
+            angularDamping: 0,
+            color: {
+                red: 255,
+                blue: 255,
+                green: 255
+            },
+            intensity: 0,
+            falloffRadius: 0,
+            isSpotlight: 0,
+            exponent: 1,
+            cutoff: 10,
+            collisionless: true
+        },
+        barrelStageLeftPosition = {
+            x: -26.4579,
+            y: 4989.7456,
+            z: -23.3428
+        },
+        barrelStageRightPosition = {
+            x: -38.6957,
+            y: 4989.5752,
+            z: -23.3843
+        },
+        entityNames = [
+            "Set_Phlash_Pad_Models_Left",
+            "Set_Phlash_Pad_Models_Right",
+            "Set_Phlash_Pad_Left",
+            "Set_Phlash_Pad_Right",
+            "Set_Phlash_Pad_Zone",
+            "Set_Phlash_Particles_Left",
+            "Set_Phlash_Particles_Right",
+            "Set_Phlash_Debug-Cube",
+            "Set_Phlash_Lights_Stage_Left",
+            "Set_Phlash_Lights_Stage_Right",
+            "Set_Phlash_Lights_Back_Left",
+            "Set_Phlash_Lights_Back_Right"           
+        ];
+
+    // Helper Functions
+    function col(r, g, b) {
+        var obj = {};
+        obj.red = r;
+        obj.green = g;
+        obj.blue = b;
+        return obj;
+    }
+
+    function vec(x, y, z) {
```

Comment:
Can we use the built-in Vec3 class for this?

Reply:
:( not currently.  have to mod it to be able to actually create a vec3 object from arguments.  It's just such a flow helper not having to make those objects manually when you are doing a lot of them.